### PR TITLE
br: add foreign key test for br

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary: add foreign key test for br

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by YuJuncen — APPROVED** [review]

File: br/pkg/gluetidb/glue.go [hunk 1]
```diff
@@ -231,6 +231,8 @@ func (gs *tidbSession) CreateTables(ctx context.Context, tables map[string][]*mo
 			cloneTables = append(cloneTables, table)
 		}
 		gs.se.SetValue(sessionctx.QueryString, queryBuilder.String())
+		// Disable foreign key check when batch create tables.
+		gs.se.GetSessionVars().ForeignKeyChecks = false
```
Comment (fengou1) [inline br/pkg/gluetidb/glue.go:235]: it looks like we can set ForeignKeyChecks out of this loop.  my understanding is that ForeignKeyChecks is per session. only need to set it per session will be enough? correct me if I am wrong @crazycs520

Comment (crazycs520) [reply br/pkg/gluetidb/glue.go]: great, done.

**Review by fengou1 — APPROVED** [review]

**Pull request merged**